deploy/examples/cloud_resources_aws_strategies.yaml

```diff
@@ -9,3 +9,6 @@ data:
     {"development": { "region": "eu-west-1", "createStrategy": {}, "deleteStrategy": {} }}
   postgres: |
     {"development": { "region": "eu-west-1", "createStrategy": {}, "deleteStrategy": {} }}
+  _network: |
```

Comment:
What is the reason for the underscore in front of the network name?

Reply:
@dimitraz The idea here is that `_network` doesn't define settings for a resource that users can create or update. It's configurable, but it's used internally in CRO.

It also won't follow the same pattern as other keys in this configmap. We won't really reconcile on this, we'll create the VPC with the provided CIDR and that's pretty much it. If you change it, it'll be ignored. The delete strategy has just been added to allow for extensibility.

Reply:
Ok that makes sense. Would it be worth documenting this somewhere?

Reply:
Good point. @ciaranRoche Mind throwing a note about this into https://github.com/integr8ly/cloud-resource-operator/tree/master/doc ?

Reply:
@aidenkeating keeping note of all documentation, i was thinking of holding off for a few days before capturing documentation, just in case anything changes. Ill add this to my list and follow up after make some more progress

Reply:
@ciaranRoche I think it's best to capture documentation when we can do it piece by piece instead of in a rush. Small refactors of documentation aren't that bad compared to documenting an entire solution at the end.

`_network` isn't going to change from being a CreateVPCInput struct.